modules/crud/operations.go

```diff
@@ -232,6 +232,19 @@ func (m *Module) CreateProjectIfNotExists(ctx context.Context, project, dbType s
 	return crud.RawExec(ctx, sql)
 }
 
+// GetConnectionState gets the current state of client
+func (m *Module) GetConnectionState(ctx context.Context, dbType string) (bool, error) {
+	m.RLock()
+	defer m.RUnlock()
+
+	crud, err := m.getCrudBlock(dbType)
+	if err != nil {
+		return false, err
```

Comment:
do not return error here, only return false 
so change the GetconnetionState function return values
now it will only return bool